_posts/2024-11-20-cockpit-329.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 329
+author: mvo
+date: '2024-11-20'
+tags: cockpit, files
+slug: cockpit-329
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from cockpit 329 and cockpit-files 12:
+
+
+## Shell: Extra warnings when connecting to remote hosts
+
+Connecting to multiple hosts in a single Cockpit session allows all
+these hosts to access each other freely. Eventually Cockpit will not
+allow multiple connections, but in the mean time, we want to educate
+people better.
+
+![screenshot of extra warnings when connecting to remote hosts]({{ site.baseurl }}/images/329-shell-extra-warnings-when-connecting-to-remote-hosts.png)
```

Comment:
That screenshot  is out of date, no? I remember we replaced "JavaScript" with something something "malicious actions".

Reply:
Yes, true. Fixed by using the pixel test reference.

Reply:
Ouch, that doesn't have a border...

Reply:
Fixed.